### PR TITLE
added custom blurb code

### PR DESCRIPTION
**Pull request by ivanskof**

### CODE REVIEW

File: lib/jekyll-multiple-languages-plugin.rb [hunk 1]
```diff
@@ -307,16 +311,44 @@ def render(context)
...
-        puts "Using translation '%s' from default language: %s" %[translation, site.config['default_lang']]
+
+         # Custom Blurb code: don't show warnings for EN only files
+         # Custom Blurb code: and don't show warnings for en-ca and en-au
+         dont_show_warnings_for_languages = site.config['dont_show_warnings_for_languages']
+         if (dont_show_warnings_for_languages && !dont_show_warnings_for_languages.include?(lang)) or lang == site.config['default_lang']
+           puts "Missing i18n key: #{lang}:#{key}".yellow
+         end
+
+        # Custom Blurb code: Removed to reduce noise
```
Comment (chicagoing) [inline lib/jekyll-multiple-languages-plugin.rb]: You can remove lines 350 & 351.

File: lib/jekyll-multiple-languages-plugin.rb [hunk 2]
```diff
@@ -72,7 +72,10 @@ def process
       self.parsed_translations ||= {}
       
       self.config['exclude_from_localizations'] ||= []
-      
+
+      # Custom Blurb code
+      self.config['default_lang_only'] ||= []
```
Comment (chicagoing) [inline lib/jekyll-multiple-languages-plugin.rb]: I don't see this being used anywhere. Should we remove it for now?

**Review by chicagoing — CHANGES_REQUESTED** [review]

**Comment by ivanskof** [comment]
@chicagoing changes done

**Review by chicagoing — APPROVED** [review]
+1

**Comment by chicagoing** [comment]
Looks good. Merge when you're ready. Thank you.

**Comment by ivanskof** [comment]
@lpeacock @chicagoing I can't merge, I don't have write access.

**Pull request merged**